### ISSUE TITLE
ColorPicker: No support for adding aria text to the text boxes

### ISSUE THREAD

**Issue by anjali2408** [OfficeDev/office-ui-fabric-react#4940]
### Bug Report

- __Package version(s)__: 5.76.0

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low) Normal

Products/sites affected: (if applicable) Power BI Visio Visual

#### Describe the issue:

Colorpicker: No support for adding aria text to the color text boxes. The user will not know about what text field they are editing. Basically there should be an option to add redLabel, blueLabel, greenLabel, hexLabel and alphaLabel as an aria label so that the narrator can read it.

#### Actual behavior:

Narrator reads the value inside the textbox along with editing. eg: if the focus is on the redLabel textbox and the value in the textbox is 45 then the Narrator reads "45 editing"

#### Expected behavior:

Narrator should read the text box label along with the value. eg: if the focus is on the redLabel textbox and the value in the textbox is 45 then the Narrator reads "redLabel 45 editing"
